lib/gps.js

```diff
@@ -432,6 +562,40 @@ GPS.prototype.parseNmeaSentence = function(sentence) {
 
 };
 
+/*
+ * SkyTraq Binary Message Protocol
+ * http://cdn.sparkfun.com/datasheets/Sensors/GPS/Venus/638/doc/AN0003_v1.4.19.pdf
+ */
+GPS.prototype.parseSkyTraqSentence = function(sentence) {
```

Comment:
But by putting them here, they are defined for all GPS instances, which I don't want. I think that the whole GPS class needs to be re-evaluated such that it can handle disparate sources in a uniform way. 

> Is the issue just with having the utility functions available?

I don't want component-specific things on prototypes. That's why we have controller and breakout definitions, but I think something isn't lining up in GPS. I want @dtex to take a look at this and also @BrianGenisio